src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.input {
+  margin-bottom: 10px;
+}
```

Comment:
```suggestion
.input:not(:last-child) {
  margin-bottom: 10px;
}
```
It may fix your tests

Reply:
Ok. Tests are passed. Thanks.